### PR TITLE
Adding data and changeLog.yml for data dir

### PR DESCRIPTION
**Pull request by mhines-usgs**
- [x] agency - source devnolog agency
- [x] aquifer  - source devnolog aquifer
- [x] national aquifer  - source devnolog nat aqfr
- [x] country  - source devnolog  - source devnolog country
- [x] state  - source devnolog  - source devnolog state minus unspecified 00
- [x] county  - source devnolog  - source devnolog county minus unspecified 000
- [x] minor civil division  - source devnolog mcd
- [x] hydrologic unit - source devnolog huc
- [x] horizontalAccuracyCode - source https://pubs.usgs.gov/of/2004/1238/pdf/gwsi_complete.pdf page 18, 1.18 Lat-Long Coordinate Accuracy
- [x] horizontalDatum - source http://www.epsg-registry.org/, based on swagger comment for CRS84
- [x] horizontalMethodCode - source https://pubs.usgs.gov/of/2004/1238/pdf/gwsi_complete.pdf page 19, 1.19 Lat/Long Method
- [x] altitudeDatum - source swagger and https://en.wikipedia.org/wiki/Template:NGVD29 for names
- [x] altitudeMethodCode - source  https://pubs.usgs.gov/of/2004/1238/pdf/gwsi_complete.pdf page 20, 1.24 Altitude Datum 
- [ ] use
- [ ] monitoring location type

### CODE REVIEW

**Comment by mhines-usgs** [comment]
If this looks ok @kmschoep-usgs  I'll keep adding to the changeLog.yml and the csv files that I can harvest from natdb

**Comment by kmschoep-usgs** [comment]
Looks good!

**Comment by mhines-usgs** [comment]
Not sure about use and monitoring location type... will wait until I can chat with the rest of team.

File: mlr-database-schema/src/main/liquibase/data/horizontal_method.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
+"code","name"
+"C","Calculated from land net"
+"D","Differentially corrected Global Positioning System (GPS)"
+"G","Global positioning system, uncorrected (Standard Positioning Service (SPS)"
+and Precise Positioning Service (PPS))"
```
Comment (kmschoep-usgs) [inline mlr-database-schema/src/main/liquibase/data/horizontal_method.csv]: this has a line break in there that will probably cause issues.

**Comment by mhines-usgs** [comment]
@kmschoep-usgs Good catch, thanks! I remember seeing the end of that line and thinking, hey why isn't there a double quote, and adding one, but I didn't notice that the next line wasn't what it was supposed to be. I'll try to get this to run locally today...

**Pull request merged**